install_config/install/planning.adoc

```diff
@@ -52,27 +52,50 @@ If you install initially using the quick installer, you can always further tweak
 [[sizing]]
 == Sizing Considerations
 Determine how many nodes and pods you require for your {product-title} cluster.
-The following table provides the sizing limits for nodes and pods:
+Cluster scalability correlates to the number of pods in a cluster environment.
+That number drives everything else.
+
+The following table provides the maximum sizing limits for nodes and pods:
 
 [cols="8,2",options="header"]
 |===
 |Type |Maximum
 
 |Maximum nodes per cluster |1000
 
-|Maximum pods per cluster |120,000
+|Maximum pods per cluster |120000
```

Comment:
http://stylepedia.net/#chap-Red_Hat_Technical_Publications-Writing_Style_Guide-N

See "numbers" for comma usage.